scripts/jenkinsProductionFiles.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# jenkinsProductionFiles.sh by Andrzej Pogonowski
+
+# Steps
+# 1. Removes all of the build and dependency directories (build, pack, node_modules)
+# 2. Copies all of the source files
+# 3. Copes configuration files and other necessary files
+
+# Cleanup
+rm -r pack
+rm -r build
+rm -r node_modules
+mkdir pack
+chmod -R 755 pack
+
+# Copy needed directories
+cp -rf ./src ./pack/src
+cp -rf ./cypress ./pack/cypress
+cp -rf ./.storybook ./pack/.storybook
```

Comment:
Is the storybook directory needed for production code?

Reply:
you guys deploy storybook to github pages right?

Reply:
Storybook is now deployed under the simorgh infra pipeline, so yes this dir would be needed. I don't like the idea of storybook being in the final RPM though....

Reply:
Ohhh that's annoying, forgot about the github pages deploy

Reply:
@hindsc52 I will add this as an issue.